website/snippets/_bigquery-dataproc.md

```diff
@@ -0,0 +1,3 @@
+To run dbt Python models on GCP, dbt uses companion services, Dataproc and Cloud Storage, that offer tight integrations with BigQuery. You may use an existing Dataproc cluster and Cloud Storage bucket, or create new ones:
```

Comment:
i love that you used a snippet here! but it looks like you're using it only on one page: the dbt cloud bigquery page. did you mean to use it only for one page or for the [core](https://docs.getdbt.com/docs/core/connect-data-platform/bigquery-setup#running-python-models-on-dataproc) AND cloud bq page?